pages/reference-architecture/_sub-hero.html

```diff
@@ -3,10 +3,15 @@
     <h2>A new standard for architecture on AWS</h2>
     <p>
       The Reference Architecture is an opinionated, battle-tested, best practices way to assemble the code from the
-      <a href="/gruntwork-library">Gruntwork Library</a> into an end-to-end tech stack that has just about everything
-      you need: server cluster, load balancer, database, cache, network topology, monitoring, alerting, CI/CD, secrets
-      management, VPN, and more. We can deploy the Reference Architecture into your AWS account(s), give you 100% of
-      the code, and do a walkthrough training, all in about a day.
+      <a href="/infrastructure-as-code-library">Infrastructure as Code Library</a> into an end-to-end tech stack that
+      has just about everything you need: server cluster, load balancer, database, cache, network topology, monitoring,
+      alerting, CI/CD, secrets management, VPN, and more.
+    </p>
+    <p>
+      The Reference Architecture is highly customizable. You get to specify the type of server cluster, load balancer,
+      database, cache, CI server, network settings, DNS settings, service discovery mechanism, and so on, from a list
+      of supported options. We create the architecture to your specifications, deploy it into your AWS account(s), and
+      give you 100% of the code, all in about a day.
```

Comment:
Change to:

```html
<h4>The Reference Architecture is highly customizable.</h4>
<p>You get to specify the type of server cluster, load balancer...</p>
```

Reply:
Looks a bit weird with lots of bright headers next to each other.